src/helpers/util/createPositionalHelpers.js

```diff
@@ -0,0 +1,25 @@
+//@flow
+import Style from "further";
+
+import upperCaseFirstLetter from "./upperCaseFirstLetter";
+import lowerCaseFirstLetter from "./lowerCaseFirstLetter";
+
+type IHelperDescriptor = {
+  style: Function,
+  name: string,
+  place: string,
+};
+export const positions = ["left", "right", "top", "bottom"];
+
+export default (
```

Comment:
This could be merged with `createSizeHelpers` with a little work. They both take an array and convert them into an array containing `{ name: string, style: Style }` which is used to attach helpers.

I propose merging them into `createHelpers` which takes the following shape

```js
type IHelperDescription = {
  name: string,
  lens: string => props => style,
}

type IHelperList = {
  name: string // the name of the helper (i.e "left", "sm")
  ?propertyModifer: string  => string // the name of the css property to modify (i.e margin => "marginLeft")
  ?reducer: any => any // the reducer for the previous value (i.e for "sm" it would be x => x/d)
}[]

type IHelperResult = {
  name: string,
  style: Style,
}[]
// .createHelper :: IHelperDescription => IHelperList => IHelperResult

```